msteams-platform/bots/how-to/format-ai-bot-messages.md

```diff
@@ -0,0 +1,459 @@
+---
+title: Format AI bot messages
+description: Learn how to format your AI-based bot with an AI label, sensitivity labels, citations, and feedback buttons.
+ms.topic: conceptual
+ms.localizationpriority: medium
+---
+
+# Format AI bot messages
+
+If you're building a conversational bot, you can now leverage features such as citations and feedback buttons that enable better user engagement. For AI-powered bots, you can now add AI label and sensitivity label that can help you build trust and transparency with users.
+
+Here's a quick overview on each feature that you can integrate to your bot message:
+
+| Feature | Bot | AI bot |
+| --- | :---: | :---: |
+| [AI label](#ai-label) that enables user to identify that the message was generated using AI. | ❌ | ✔️ |
+| [Sensitivity label](#sensitivity-label) that enables user to understand the confidentiality of the bot's message. | ❌ | ✔️ |
+| [Citations](#citations) that enables user to refer to the source of the bot's message through in-text citations and a list of references. | ✔️ | ✔️ |
+| [Feedback buttons](#feedback-buttons) that enables user to provide positive or negative feedback based on their experience. | ✔️ | ✔️ |
+
+:::image type="content" source="../../assets/images/bots/ai-bot-message.png" border="false" alt-text="Screenshot shows the AI-generated bot message with the respective AI bot message features.":::
+
+> [!NOTE]
+>
+> AI label, sensitivity label, citations, and feedback buttons:
+>
+> * Are available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+> * Aren't supported in message extensions.
+
+## AI label
+
+AI label added to your bot message specifies that the message is generated by AI. AI bots use large language models (LLMs) that are generally reliable, however, it's crucial to add an AI label to maintain user's trust on your bot.
+
+To add an AI label to your bot message, modify the message to include `additionalType` field under the `entities` object.
+
+The following code snippet shows how to add an AI label to your bot's message:
+
+```json
+{ 
+    "type": "message", 
+    "from": { 
+        "id": "28:48b2e1fd-b6b8-46ce-8074-8e918304fcb2" 
+    }, 
+    "conversation": { 
+        "isGroup": true, 
+        "id": "19:88b15596-2dd4-440b-98c6-a19cf49455a1@thread.tacv2" 
+    }, 
+    "recipient": { 
+        "id": "8:orgid:f20c2a2c-1b43-416a-b5e8-fbcc69387c5e" 
+    }, 
+    "text": "2fd50566-ad63-4237-b240-f1942f7bdeb6", 
+    "attachments": [], 
+    "entities": [ 
+        { 
+            "@type": "Message", 
+            "@context": "https://schema.org", 
+            "@id": "", 
+            "additionalType": [ 
+                "AIGeneratedContent" 
+            ] 
+        } 
```

Comment:
Isn't the entire object (including @type, etc) required?

Reply:
**@corinagum**, we are unsure what changes are to be made here. Could you specify what needs to be modified?

Reply:
Above the documentation says to add AIGeneratedContent, all properties in the object should be added.

Reply:
Understood @corinagum, it is updated now with a table defining whether the properties are optional or not.